pom.xml

```diff
@@ -19,7 +19,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <version.logback>1.3.3</version.logback>
     <version.jackson>2.13.4</version.jackson>
-    <version.cdcsdk>1.0.0.RC1</version.cdcsdk>
+    <version.cdcsdk>1.0.0.RC7</version.cdcsdk>
```

Comment:
Let's make sure that we have the `cdcsdk-server` jar released before merging this PR.

Reply:
Yes the requirred jar has been released